Fix for sparc.
This should be investigated for sending upstream. 
--- a/mail/installer/package-manifest.in
+++ b/mail/installer/package-manifest.in
@@ -564,6 +564,7 @@
 @BINPATH@/@DLL_PREFIX@freebl_32fpu_3@DLL_SUFFIX@
 @BINPATH@/@DLL_PREFIX@freebl_32int_3@DLL_SUFFIX@
 @BINPATH@/@DLL_PREFIX@freebl_32int64_3@DLL_SUFFIX@
+@BINPATH@/cpu/sparcv8plus/@DLL_PREFIX@nspr_flt4@DLL_SUFFIX@
 #endif
 
 ; core platform files
